src/status_im/contexts/wallet/add_account/add_address_to_watch/confirm_address/view.cljs

```diff
@@ -38,8 +38,7 @@
                                :disabled?           (string/blank? @account-name)
                                :accessibility-label :confirm-button-label
                                :on-press            #(rf/dispatch [:wallet/add-account
-                                                                   {:sha3-pwd     nil
-                                                                    :type         :watch
+                                                                   {:type         :watch
```

Comment:
Just removed the `sha3-pwd` that is passing `nil`. It's not needed and it's not a good practice because the `:or` destructuring (if present) may be ignored.